src/stories/DatePicker.stories.tsx

```diff
@@ -116,6 +126,28 @@ stories.add("Disabled Before Today", () => {
   );
 });
 
+stories.add("Disabled Before Date and Disabled after Date", () => {
+  return (
+    <DatePicker
+      numberOfMonths={2}
+      disabledBeforeDate={text("disabled before", disabledBeforeDate)}
+      disabledAfterDate={text("disabled after", disabledAfterDate)}
```

Comment:
Please add these props to the `All Props` story for date picker and range picker.